src/KM_Controls.pas

```diff
@@ -5311,6 +5313,9 @@ procedure TKMColumnBox.UpdateItemIndex(Shift: TShiftState);
   begin
     fTimeOfLastClick := 0; //Double click shouldn't happen if you click on one server A, then server B
     ItemIndex := NewIndex;
+    if Assigned(fOnChange)
+      and not KMSamePoint(fMouseOverCell, INVALID_MAP_POINT) and Columns[fMouseOverCell.X].TriggerOnChange then

```

Comment:
Please swap conditions around

if not KMSamePoint(fMouseOverCell, INVALID_MAP_POINT) and Columns[fMouseOverCell.X].TriggerOnChange
and Assigned(fOnChange) then